packages/moonstone/styles/variables.less

```diff
@@ -148,23 +148,36 @@
 @moon-button-height-large: 96px;
 @moon-button-small-height: 60px;
 @moon-button-small-height-large: 72px;
-@moon-button-border-width: 6px;
-@moon-button-border-radius: 9999px;
+@moon-button-border-width: 3px;
+@moon-button-border-radius: 3px;
+// @moon-button-border-radius: 960px;
```

Comment:
It should be removed. I realized during construction that 999 is a dumb number to use to round corners, while 960 is nice because it translates to an even 40rem. I had written a comment as such, but I removed it at some point. This can just be deleted as cruft.